enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult/InteractiveQuestionResult.tsx

```diff
@@ -49,20 +52,35 @@ export const InteractiveQuestionResult = ({
     onSave,
     isSaveEnabled,
     saveToCollectionId,
+    isCardIdError,
   } = useInteractiveQuestionContext();
 
   const [isSaveModalOpen, { open: openSaveModal, close: closeSaveModal }] =
     useDisclosure(false);
 
   // When visualizing a question for the first time, there is no query result yet.
-  const isQueryResultLoading = question && !queryResults;
+  const isQueryResultLoading =
+    question && shouldRunCardQuery(question) && !queryResults;
 
   if (isQuestionLoading || isQueryResultLoading) {
     return <SdkLoader />;
   }
 
-  if (!question) {
-    return <SdkError message={t`Question not found`} />;
+  // `isCardError: true` when the entity ID couldn't be resolved
+  if (!question || isCardIdError) {
+    return (
+      <SdkError
```

Comment:
There's no design for this component, so the design could be changed, I use `Code` directly from Mantine, but if we're really going to use it I will add it to `metabase/ui`